Mathlib/Data/Int/Basic.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2016 Jeremy Avigad. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Avigad
+-/
+import Mathlib.Data.Int.Init
+import Mathlib.Data.Nat.Basic
+import Mathlib.Logic.Nontrivial.Defs
+import Mathlib.Tactic.Convert
+import Mathlib.Tactic.Lift
+import Mathlib.Tactic.OfNat
+
+/-!
+# Basic operations on the integers
+
+This file builds on `Data.Int.Init` by adding basic lemmas on natural numbers
```

Comment:
```suggestion
This file builds on `Data.Int.Init` by adding basic lemmas on natural numbers
```
Did you mean to say "natural numbers" there?

Reply:
Not at all, this is a copypasta error. :)
```suggestion
This file builds on `Data.Int.Init` by adding basic lemmas on integers.
```